core/package.json

```diff
@@ -66,7 +66,7 @@
     "jest-cli": "^27.5.1",
     "prettier": "^2.6.1",
     "rollup": "^2.26.4",
-    "sass": "^1.26.10",
+    "sass": "^1.33.0",
```

Comment:
Prior to upgrading `sass` I was getting this error when building with the `dynamic-font` function:

<img width="662" alt="Screenshot 2023-08-22 at 4 04 52 PM" src="https://github.com/ionic-team/ionic-framework/assets/6577830/3cbbaf01-b4d8-47a9-983d-f007cfebd800">

I found this [Stack Overflow thread](https://stackoverflow.com/questions/67857768/sass-math-div-function-is-undefined-in-angular-app-but-is-defined-in-angular-li) which mentions upgrading to `1.33.0` or higher and that fixed the problem so this change is required for the build to work.